### DISCUSSION TITLE
GitHub Education Application Not Being Approved

### DISCUSSION THREAD

**Discussion by Betanoir** [community/community#58519] [GitHub Education]
### Hi everyone,

Hi there, 

I have applied three times in the past couple days to attempt to gain access to GitHub Education and I have been rejected each time.

The reasons I get are as follows:

```
Your application cannot be reviewed until you fix the following:

- You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
- The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
- You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.
```

I have completed step 1; however, my college doesn't seem to want to be accepted as an institution in my country and I am not sure why. I do live in the same country as my college and I attend it physically.

I have provided the printed evidence from my college principle stating that I attend this whole year and the application still fails to work.

What could be the issue?

Many thanks!

**Comment by samus-aran** [comment]
Hi @Betanoir 

There are many similarly named schools across the world. If you are unable to find your school on the application map then I suggest you submit the school details so we can verify the institution. You can do this by not selecting a school and enter the school name. I may also suggest you append to it the campus so it differentiates from the school name that is located elsewhere.

Hope this helps 👍 Let us know how you get along.
